contribs/gmf/src/directives/layertree.js

```diff
@@ -578,6 +574,26 @@ gmf.LayertreeController.prototype.toggleActive = function(treeCtrl) {
 
 
 /**
+ * LayertreeController.prototype.getLayersNames_ - Get the layer(s) name(s)
+ * Attached to the node param, regarding its type
+ *
+ * @param  {GmfThemesNode} node The tree node
+ * @return {string} name(s) of the layer(s) attached to this node regarind its type
+ * @private
+ */
+gmf.LayertreeController.prototype.getLayersNames_ = function(node) {
+  var type = gmf.Themes.getNodeType(node);
+  switch (type) {
+    case gmf.Themes.NodeType.WMS:
+      return node.layers;
+    case gmf.Themes.NodeType.WMTS:
+      return node.layer;
+    default:
+      throw new Error('node wrong type to get layer(s) name(s): ' + type);
```

Comment:
Ok. should also be modified on https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/layertree.js#L295 and https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/layertree.js#L266